src/lmn-gtm-analytics.js

```diff
@@ -127,10 +128,13 @@ const lmnAnalytics = {
       options = null;
       traits = null;
     }
-    dataLayer.push(
-      Object.assign(eventMetaData(),
-      { user: { userId: id } }
-    ));
+
+    Object.assign(traits, eventMetaData());
+    // reconstruct arguments after sanitisation to proxy through
+    const args = [id, traits, options, callback].filter(x => x);
+    analytics.identify.apply(this, args);
+
+    dataLayer.push(Object.assign(traits, { user: { userId: id } }));
```

Comment:
You could probably avoid using multiple `Object.assign`

```js
dataLayer.push({
  ...traits,
  ...eventMetaData(),
  user: { userId: id }
});
```

#ES6destructurem8